include/aspect/newton.h

```diff
@@ -105,6 +105,17 @@ namespace aspect
        */
       void set_newton_derivative_scaling_factor(const double newton_derivative_scaling_factor);
 
+      /**
+       * Gets the Newton derivative scaling factor used for scaling the
+       * derivative part of the Newton Stokes solver in the assembly.
+       */
+      std::pair<std::string,std::string> get_Newton_stabilisation() const;
```

Comment:
`stabilization` please. Otherwise we end up in an American vs. British English war :smile:.

Reply:
After reading through the rest of the PR, I think it would be better to split both functions into separate functions that could be named `set_preconditioner_stabilization` and `set_velocity_block_stabilization` respectively (and the same for the `get_...` functions. This is more intuitive than returning or setting a pair of parameters. Otherwise it would get real complicated if somebody added a third stabilization option.